subsys/logging/log_output.c

```diff
@@ -114,18 +115,23 @@ static int print_formatted(const struct log_output *log_output,
 	return length;
 }
 
-void log_output_flush(const struct log_output *log_output)
+static void buffer_flush(log_output_func_t outf, u8_t *buf, size_t len,
```

Comment:
I find it strange that function which performs "write" operation is called "flush". "Flush" operation generally doesn't take params (except for maybe timeout), and flushes whatever already accumulated.

Reply:
yes, i did rename only function prefix after extracting it from `log_output_flush`. changed to `buffer_write`.